numpy/lib/npyio.py

```diff
@@ -1717,6 +1717,16 @@ def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
     array((1, 1.3, b'abcde'),
           dtype=[('intvar', '<i8'), ('fltvar', '<f8'), ('strvar', 'S5')])
 
+    An example to show comments
+
+    >>> f = StringIO('''
+    ... text,# of chars
```

Comment:
Since the comments param is '#', it gets discarded from output in the example. Should that be made more explicit since there is already a docstring for comments?

Reply:
Right, but the original question was "what do I do if I do _not_ want the # to be treated as a comment